samples/electrophoresis.py

```diff
@@ -17,30 +17,26 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 """
-This sample simulates electrophoresis using P3M solver.
+Simulate electrophoresis of a linear polymer using the P3M solver.
 """
 import espressomd
 
 required_features = ["P3M", "EXTERNAL_FORCES", "WCA"]
 espressomd.assert_features(required_features)
 
-from espressomd import thermostat
 from espressomd import interactions
 from espressomd import electrostatics
-import sys
 import numpy as np
 try:
     import cPickle as pickle
 except ImportError:
     import pickle
```

Comment:
I think we should remove pickle here and also the lattice Boltzmann stuff. Either we show a sample with hydrodynamics and explain it or we don't. Checkpointing is also not part of this sample.

Reply:
agreed, let's remove pickle and lb